crates/pevm/tests/ethereum/main.rs

```diff
@@ -164,24 +164,31 @@ fn run_test_unit(path: &Path, unit: TestUnit) {
                 (Some("TR_TypeNotSupported"), Ok(_)) => {}
                 // Remaining tests that expect execution to fail -> match error
                 (Some(exception), Err(PevmError::ExecutionError(error))) => {
+                    use revm::primitives::InvalidTransaction as T;
```

Comment:
nit: personally, I am against using `as T` here. `InvalidTransaction::` is clearer than `T::`.